### PR TITLE
Add_mongo_mount

### PR DESCRIPTION
**Pull request by jljalderman**
Add additional mount point for EBS volume for mongo data

### CODE REVIEW

File: hieradata_aws/class/mongo.yaml [hunk 1]
```diff
@@ -10,3 +10,15 @@ mongodb::server::replicaset_members:
   'mongo-1':
   'mongo-2':
   'mongo-3':
+
+lv:
+  data:
+    pv:
+      - /dev/sdf
```
Comment (surminus) [inline hieradata_aws/class/mongo.yaml]: These machines are `m5.large`, which means they use a different device label. Check other hieradata to see what it should be (as I can't remember off the top of my head).

Comment (jljalderman) [reply hieradata_aws/class/mongo.yaml]: https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/device_naming.html and https://aws.amazon.com/amazon-linux-ami/instance-type-matrix/ suggest that M5.large should be HVM EBS backed 64bit, which it suggests for EBS volumes should be /dev/sd[f-p]

Comment (surminus) [reply hieradata_aws/class/mongo.yaml]: [Related commit I was talking about](https://github.com/alphagov/govuk-puppet/commit/b1e5add07092de506bee9f0322840fba99acc004)

Comment (jljalderman) [reply hieradata_aws/class/mongo.yaml]: I did see that, I don't know the answer but happy to change it...

Comment (jljalderman) [reply hieradata_aws/class/mongo.yaml]: done

**Pull request closed without merging**